lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1437,6 +1437,22 @@ OpFoldResult AtenBoolIntOp::fold(FoldAdaptor adaptor) {
   return nullptr;
 }
 
+//===----------------------------------------------------------------------===//
+// AtenAnyBoolOp
+//===----------------------------------------------------------------------===//
+
+OpFoldResult AtenAnyBoolOp::fold(FoldAdaptor adaptor) {
+  auto inputConsturct = getSelf().getDefiningOp<Torch::PrimListConstructOp>();
```

Comment:
Drive by comment: need to check that the defining op is indeed a `PrimListConstructOp` and that the list is not mutated. Similar to this: 

https://github.com/llvm/torch-mlir/blob/139e820e15fc441fe90f595ee927035304b5b606/lib/Dialect/Torch/IR/TorchOps.cpp#L1931-L1933